src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -2,18 +2,143 @@
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+
+    private Node<K, V>[] table;
+    private int size;
+    private int capacity;
+    private final float loadFactor;
+    private int threshold;
```

Comment:
please remove these 3 fields, code will be simpler without them
capacity is always a size of the table, loadfactor never changes, and we have constant for it
threshold is always a product of the two above